scripts/build-local-release.sh

```
@@ -0,0 +1,125 @@
+  echo "==> Downloading GhosttyKit..."
+  bash scripts/download-prebuilt-ghosttykit.sh
+fi
+
+# --- Remove problematic xcframework tags from ghostty submodule ---
+for tag in $(git -C ghostty tag -l 'xcframework-*' 2>/dev/null); do
+  git -C ghostty tag -d "$tag" >/dev/null 2>&1 || true
+done
+
+# --- Build Release ---
+echo "==> Building Release..."
+rm -rf "$BUILD_DIR/"
```

Shellcheck (0.11.0)

[warning] Use "${var:?}" to ensure this never expands to / .

(SC2115)